src/Scenario/UserContactsScenario.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TelegramOSINT\Scenario;
+
+use TelegramOSINT\Client\InfoObtainingClient\Models\UserInfoModel;
+use TelegramOSINT\Exception\TGException;
+use TelegramOSINT\Logger\Logger;
+
+class UserContactsScenario extends InfoClientScenario
+{
+    /** @var callable|null */
+    private $cb;
+    /** @var callable[] */
+    private $callQueue = [];
+
+    /**
+     * @param callable|null                 $cb              function()
+     * @param ClientGeneratorInterface|null $clientGenerator
+     *
+     * @throws TGException
+     */
+    public function __construct(?callable $cb = null, ClientGeneratorInterface $clientGenerator = null)
+    {
+        parent::__construct($clientGenerator);
+        $this->cb = $cb;
+    }
+
+    /**
+     * @throws TGException
+     */
+    protected function getContactsInfo(): void
+    {
+        $phones = [
+            '+79153801634',
+        ];
```

Comment:
These numbers shoul not be hardcoded

Reply:
right, removed those, now example awaits number list in argv.